docs/slow-network-test.rst

```diff
@@ -0,0 +1,62 @@
+Testing Raiden on a slow network
+================================
+
+In order to analyze Raiden performance on slow networks, it is useful to have a way of
```

Comment:
Can you please add a sentence that this is linux specific and also mention which tools / packages the example script requires?

Reply:
Done.